src/platform/windows/window.rs

```diff
@@ -280,11 +279,16 @@ impl Window {
     }
 
     #[inline]
-    pub fn set_maximized(&self, maximized: bool) {
+    pub fn set_maximized(&self, _maximized: bool) {
```

Comment:
Should contain `unimplemented!()`.

Reply:
I didn't set these as unimplemented because they're safe as no-ops. No need to panic the app just because we can't set it as maximized/fullscreen.

Reply:
Using no-ops means that these functions will likely be left unimplemented for years.

Reply:
Ok, done now